tools/bin/commands/release

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+NEXUS_URL="https://oss.sonatype.org"
+NEXUS_STAGING_PLUGIN_VERSION="1.6.7"
+NEXUS_SERVER_ID="oss-sonatype-staging"
+NEXUS_STAGING_REPO_PREFIX="iosyndesis" #This is the parent groupId with dots removed.
+NEXUS_DEFAULT_STAGING_PROFILE_ID="61015c63f807ce" #This this is the id of the staging profile. Just go to the list of staging profiles, click the one u need and grab that id from the url (its not displayed in the actual list).
```

Comment:
does it make sense to make this default and allow override via command line switch?

Reply:
Not sure. I don't think this is something that will ever change.